### PR TITLE
add @fontsource/roboto and fixed compile errors

### PR DESCRIPTION
**Pull request by jonathanlei**
## High Level Overview of Change

switched from typeface-roboto  to @fontsource/roboto

### Context of Change

issue: https://github.com/ripple/explorer/issues/172

### Type of Change

- [X] Refactor (non-breaking change that only restructures code)
- [X] Bug fix (non-breaking change which fixes an issue)

## Before / After
changed from typeface-roboto  to @fontsource/roboto in dependency. 
fixed minor compiling issues

### CODE REVIEW

**Review by ckniffen — APPROVED** [review]

**Review by intelliot — APPROVED** [review]

File: src/containers/shared/hooks.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import { useTranslation } from 'react-i18next'
 
-export const useLanguage = () => useTranslation().i18n.resolvedLanguage
+export const useLanguage = () => useTranslation().i18n.resolvedLanguage;
```
Comment (ckniffen) [inline src/containers/shared/hooks.ts:3]: after @ledhed2222 commit this should not have a colon.  I don't know why it passes linting though.

File: src/index.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ import './containers/shared/css/global.css'
 import App from './containers/App'
 import i18n from './i18n'
 
-require('typeface-roboto')
+require('@fontsource/roboto');
```
Comment (ckniffen) [inline src/index.js:16]: same

File: src/containers/shared/components/TxDetails.tsx [hunk 3]
```diff
@@ -417,7 +417,8 @@ const TxDetails = (props: Props) => {
 
   const DetailComponent = transactionTypes[type]?.TableDetail
   if (DetailComponent) {
-    return <DetailComponent instructions={instructions} />
+
+    return <DetailComponent instructions={instructions} />;
```
Comment (ckniffen) [inline src/containers/shared/components/TxDetails.tsx:421]: same

**Review by ckniffen — APPROVED** [review]

File: src/containers/shared/hooks.ts (hunk 1 above)
Comment (ckniffen) [reply src/containers/shared/hooks.ts:3]: Figured out why this doesn't fail the PR.

The PR runs `lint:js` which has `--fix`.  This causes these issues to be fixed but also means there are unstaged changes so the code does not reflect this in the repo.  One solution is to remove `--fix` the other is to do a check for unstaged changed and failed the build if those exist.  Thoughts @mvadari @ledhed2222 .

Comment (mvadari) [reply src/containers/shared/hooks.ts:3]: We should add a new `lint:ci` script that doesn't call `--fix`.

Comment (ledhed2222) [reply src/containers/shared/hooks.ts:3]: i agree that the linter shouldn't fix by default. i would prefer just no `--fix` and for users to call `npm run lint -- --fix` on their own

Comment (ledhed2222) [reply src/containers/shared/hooks.ts:3]: i didn't want to make that change in my previous PR to keep the footprint down

File: src/containers/shared/components/TxDetails.tsx [hunk 4]
```diff
@@ -417,6 +417,7 @@ const TxDetails = (props: Props) => {
 
   const DetailComponent = transactionTypes[type]?.TableDetail
   if (DetailComponent) {
+
```
Comment (ckniffen) [inline src/containers/shared/components/TxDetails.tsx:420]: Still an extra space

File: src/index.js [hunk 5]
```diff
@@ -13,7 +13,8 @@ import './containers/shared/css/global.css'
 import App from './containers/App'
 import i18n from './i18n'
 
-require('typeface-roboto')
+require('@fontsource/roboto')
+
```
Comment (ckniffen) [inline src/index.js:17]: Still an extra new line

**Review by khancode — APPROVED** [review]

**Pull request merged**